cmd/utils/flags.go

```diff
@@ -919,8 +920,32 @@ func setIPC(ctx *cli.Context, cfg *node.Config) {
 	}
 }
 
-// SetULC setup ULC config from file if given.
-func SetULC(ctx *cli.Context, cfg *eth.Config) {
+// SetLesConfig processes LES server/client/ultralight client parameters
+func SetLesConfig(ctx *cli.Context, cfg *eth.Config) {
+	if ctx.GlobalIsSet(SyncModeFlag.Name) {
+		cfg.SyncMode = *GlobalTextMarshaler(ctx, SyncModeFlag.Name).(*downloader.SyncMode)
+	}
+	if cfg.SyncMode != downloader.LightSync {
+		if ctx.GlobalIsSet(LightServFlag.Name) {
+			cfg.LightServ = ctx.GlobalInt(LightServFlag.Name)
+		} else if !ctx.GlobalBool(MiningEnabledFlag.Name) {
+			// enable light server with a given chance
+			rand.Seed(time.Now().UnixNano())
+			if rand.Intn(100) < 20 {
+				log.Warn("LES server mode enabled (use --lightserv option to explicitly enable or disable)")
```

Comment:
Could you instead specify exactly how to disable in this log message? Is it `--lightserv 0` or what?

Reply:
Could also read
```
LES server enabled by random choice. To disable this feature, use --lightserv off
```
Or something like that